enhancements/microshift/microshift-telemetry.md

```diff
@@ -0,0 +1,293 @@
+---
+title: microshift-telemetry
+authors:
+  - pacevedom
+reviewers:
+  - "ggiguash, MicroShift contributor"
+  - "pmtk, MicroShift contributor"
+  - "eslutsky, MicroShift contributor"
+  - "copejon, MicroShift contributor"
+approvers:
+  - "jerpeter1, MicroShift principal engineer"
+  - "moadz, Telemetry principal engineer"
+api-approvers:
+  - None
+creation-date: 2025-01-27
+last-updated: 2025-02-04
+tracking-link:
+  - https://issues.redhat.com/browse/OCPSTRAT-1071
+---
+
+# MicroShift telemetry
+## Summary
+MicroShift clusters are lacking the remote health monitoring that OpenShift
+has. Having no remote health monitoring means no visibility on where and how
+MicroShift has been deployed and/or is running.
+
+In order to enable visibility on the number of deployed systems and also their
+usage patterns, this enhancement proposes the addition of the
+[Telemetry API](https://rhobs-handbook.netlify.app/products/openshiftmonitoring/telemetry.md/)
+to MicroShift.
+
+## Motivation
+MicroShift is currently not sending any kind of metric from production
+deployments. Not having remote monitoring creates a blind spot on the
+deployment and usage characteristics of MicroShift instances in production
+environments.
+
+MicroShift runs as an application on top of R4E (Rhel4Edge), the usage of RHEL
+insights may yield some information about MicroShift, but there might be gaps.
+RHEL insights knows about which packages are installed, but not whether
```

Comment:
Is it useful to mention RHEL Insights as a counterpoint in the MicroShift docs?

Reply:
Is there something about this in the R4E docs?

Reply:
No, the RHEL docs have that completely separate from the install types.